src/webgpu/api/operation/vertex_state.spec.ts

```diff
@@ -0,0 +1,366 @@
+export const description = `Test indexing, index format and primitive restart.
```

Comment:
Why is the test file called vertex_state.spec.ts and not index_format.spec.ts?

Reply:
We're not really following the spec chapters, although we probably really should be (good idea) because it's hard to figure out where tests should be right now. Unfortunately the organization of the spec will change though (current plan is https://github.com/gpuweb/gpuweb/wiki/Table-of-Contents but render pipeline section organization isn't set).

Let's rename this to `vertex_state/index_format.spec.ts`? We have a bunch of other tests in the test plan doc for vertex state and they shouldn't be in the same file.

Reply:
See #392